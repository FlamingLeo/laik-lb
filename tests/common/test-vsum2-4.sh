#!/bin/sh
${LAUNCHER-./launcher} -n 4 ../../examples/vsum2 100 | LC_ALL='C' sort > test-vsum2-4.out
cmp test-vsum2-4.out "$(dirname -- "${0}")/test-vsum2-4.expected"
