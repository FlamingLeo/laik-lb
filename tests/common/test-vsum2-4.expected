Id 0: partitial sums 4950, 732, 1261, 0
Id 1: partitial sums 0, 1118, 1204, 1122
Id 2: partitial sums 0, 1332, 1228, 1584
Id 3: partitial sums 0, 1768, 1257, 2244
Total sums: 4950, 4950, 4950, 4950
